run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.TransportTracers

```diff
@@ -208,6 +209,20 @@ VerboseOnCores:              root       # Accepted values: root all
 * DELPDRY        ./GEOSChem.Restart.$YYYY$MM$DD_$HH$MNz.nc4 Met_DELPDRY         $YYYY/$MM/$DD/$HH EY   xyz 1 * - 1 1
 )))GC_RESTART
 
+#==============================================================================
+# --- O3 concentrations ---
+#==============================================================================
+(((GLOBAL_O3
+* GLOBAL_O3      $ROOT/GCClassic_Output/13.0.0/$YYYY/GEOSChem.SpeciesConc.$YYYY$MM$DD_0000z.nc4      SpeciesConc_O3    2010-2019/1-12/1/0 C xyz 1        * - 1 1
```

Comment:
@msulprizio: Should this come from the 14.0..0 10-year benchmark, as that is more recent?

Reply:
Yes, the original updates were made off of GEOS-Chem 14.0.0-rc.1 which still used 10-year benchmark output for 13.0.0. The path has now been corrected in commit 59b64930.